### DISCUSSION TITLE
Attribute error

### DISCUSSION THREAD

**Discussion by mpourasa** [Dineshkarthik/telegram_media_downloader#255] [Q&A] [answered]
Hi, I had to format my machine and start over. 

I get AttributeError: 'Client' object has no attribute 'iter_history'

Could you help me with this please?
![code](https://user-images.githubusercontent.com/61757268/174503397-b7194f30-9d8c-41f4-942d-48614b82015a.png)

**Comment by Dineshkarthik** [comment, marked as answer]
This is due to `pyrogram 2.X` installed instead of `Pyrogram 1.X` run `pip install Pyrogram==1.4.12` this should fix this issue

**Comment by mpourasa** [comment]
Problem solverd. Thank you very much!!!
